src/app/core/services/extend-query-params.service.ts

```
@@ -0,0 +1,26 @@
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+  extendQueryParamsForTextSearch(queryParams = {}, simpleSearchText: string) {
+    if (simpleSearchText === undefined || simpleSearchText.length < 1) {
+      return queryParams;
+    }
+
+    const textArray = simpleSearchText.split(/(\s+)/).filter((word) => word.trim().length > 0);
+    const lastElement = textArray[textArray.length - 1];
+    const arrayWithoutLastElement = textArray.slice(0, -1);
+
+    const searchQuery = arrayWithoutLastElement
+      .reduce((curr, agg) => agg + ' & ' + curr, '')
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)